android/src/main/java/com/reactnativeandroidwidget/builder/widget/TextWidget.java

```diff
@@ -62,6 +63,9 @@ public void applyProps() {
             view.setMaxLines(props.getInt("maxLines"));
         }
 
+        String decorationLine = props.hasKey("decorationLine") ? props.getString("decorationLine") : "none";
```

Comment:
Write something like:

```java
if (props.hasKey("textDecorationLine")) {
  String textDecorationLine = props.getString("textDecorationLine");

  if (textDecorationLine.contains("underline") {
    view.setPaintFlags(view.getPaintFlags() | Paint.UNDERLINE_TEXT_FLAG);
  }
  
  if (textDecorationLine.contains("line-through") {
    view.setPaintFlags(view.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
  }
}
```

You can put it to a method called `setDecorationLine`.
No need to check for `none` value, since by default the text does not have any decoration, so no need to modify the flags.